tests/functional/spec/repeating-household.spec.js

```diff
@@ -0,0 +1,64 @@
+import chai from 'chai'
+import {startQuestionnaire, getBlockId, getRepeatedGroup} from '../helpers'
+import AgePage from '../pages/surveys/repeating_groups/age.page.js'
+import ShoeSizePage from '../pages/surveys/repeating_groups/shoe-size.page.js'
+import HouseholdCompositionPage from '../pages/surveys/household_composition/household-composition.page.js'
+import RepeatingHouseholdPage from '../pages/surveys/repeating_household/repeating-household.page.js'
+import SummaryPage from '../pages/summary.page.js'
+
+const expect = chai.expect
+
+describe('Populating household names on subsequent repeating groups.', function() {
+
+  var repeating_household_schema = 'test_repeating_household.json';
+
+  it('Given I enter one name, when I navigate through the subsequent group, I should see the name on each block.', function() {
+    // Given
+    startQuestionnaire(repeating_household_schema)
+    var name = 'Person One'
+
+    // When
+    HouseholdCompositionPage.setPersonName(0, name).submit()
+
+    expect(RepeatingHouseholdPage.isNameDisplayed(name)).to.be.true
```

Comment:
Will chai it is better do use to.equal(name) because if/when it fails you get the comparison in the logs as debug otherwise you will just get expected true was false